<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_rtk_coorClass</class>
 <widget class="QDialog" name="dialog_rtk_coorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dialog_rtk_coor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

    QScrollBar {
        background: #1c1c1c;
        
    }

    QScrollBar:horizontal {
        height: 10px;
    }

    QScrollBar:vertical {
        width: 10px;
    }

    QScrollBar::handle {
        background:#646464;;
        border-radius: 2px;
    }
 	QScrollBar::add-page, QScrollBar::sub-page {

	    background: none;
    }

    QScrollBar::handle::pressed {
        background:#faa028;
    }

QPushButton{border:none; background:#2b2b2b} 
QPushButton{color:#ffffff} 
QPushButton::hover{color:#faa028} 
QPushButton::pressed{color:#ffffff}  


QWidget{background-color: #1c1c1c}

QLabel{color:#d9d7d7}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">QListView{
background-color:#3b3b3b;
/*color:#ffffff*/
}



QListView::item {
    color: #ffffff;
    background-color:transparent;
}

QListView::item:hover:!pressed {
    padding: -1px;
    border: 1px solid #d9d7d7;
    border-radius: 2px;
    background-color: #d9d7d7;
}

QListView::item:selected {
    padding: -1px;
    border: 1px solid #faab43;
    border-radius: 2px;
    background-color:#faab43;
}</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Choose your location:</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QPushButton" name="pushButtonNext">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <strikeout>false</strikeout>
             <stylestrategy>NoAntialias</stylestrategy>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="listViewL">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>16</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Choose your coordinate system:</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QPushButton" name="pushButtonBack">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <stylestrategy>NoAntialias</stylestrategy>
            </font>
           </property>
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="listViewC">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>16</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="pushButtonCon">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
